sdks/python/apache_beam/runners/portability/flink_runner_test.py

```diff
@@ -56,10 +56,9 @@
 
 # Run as
 #
-# pytest flink_runner_test.py \
-#     [--test_pipeline_options "--flink_job_server_jar=/path/to/job_server.jar \
-#                               --environment_type=DOCKER"] \
-#     [FlinkRunnerTest.test_method, ...]
+# pytest flink_runner_test.py[::TestClass::test_case] \
+#     --test-pipeline-options "--flink_job_server_jar=/path/to/job_server.jar \
```

Comment:
Including --flink_job_server_jar in the usage may be confusing, since it might not be obvious where the jar can be found. But if --flink_job_server_jar is omitted, it will be located automatically. Also, --environment_type=DOCKER requires the Python SDK harness container to be built. Can we either change the example, or add another, simpler example?

```
# pytest flink_runner_test.py[::TestClass::test_case] \
#     --test-pipeline-options "--environment_type=LOOPBACK"
```

Reply:
I updated it to `LOOKBACK`. Feel free to add more examples since it not easy to figure out how to run one single test locally with different configuration.

Reply:
I think this configuration will be fine for testing most things.